smartsheet-csharp-sdk/main/Smartsheet/Api/Models/AttachmentSubType.cs

```diff
@@ -23,11 +23,29 @@ namespace Smartsheet.Api.Models
     /// </summary>
     public enum AttachmentSubType
     {
+        /// <summary>
+        /// Document sub type
+        /// </summary>
         DOCUMENT,
+        /// <summary>
+        /// Spreadsheet sub type
+        /// </summary>
         SPREADSHEET,
+        /// <summary>
+        /// 
```

Comment:
nitpick: some of these are missing

Reply:
Fixed them up with latest commit please check again.

Reply:
thanks!